lib/VolSurface/Utils.pm

```diff
@@ -89,16 +89,20 @@ sub get_delta_for_strike {
         croak "Arg $_ is undef at get_delta_for_strike" unless defined $args->{$_};
     }
 
+    $new_args{forward} = $new_args{t} >= 1 ? 1 : 0;
```

Comment:
We can get rid of this line as we already included it as a parameter in the function calls of [Quant/Framework/VolSurface/Delta.pm](https://github.com/regentmarkets/perl-Quant-Framework/blob/afbbbf86e7db23354692e82f62349725e4346e0c/lib/Quant/Framework/VolSurface/Delta.pm)